src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -495,10 +663,12 @@ public void processTasks(List<Row> wbs, List<Row> tasks, List<Row> costs, List<R
          {
             task = parentTask.addTask();
          }
+         task.setProject(m_project.getProjectProperties().getName()); // P6 task always belongs to project
```

Comment:
It is; however, at least for myself I do actually use this field in client code that reads from both Project and Primavera and it's convenient for it to behave equivalently. At minimum it seems correct, if potentially superfluous.